Fix apparent misuse of the UNUSED macro; replace it with UNUSEDP. Patch sent
upstream to info-mtools@gnu.org.

--- mainloop.c.orig	Sat Feb 22 19:19:53 2014
+++ mainloop.c	Sat Feb 22 19:20:36 2014
@@ -86,7 +86,7 @@ static const char *fix_mcwd(char *ans)
 }
 
 int unix_dir_loop(Stream_t *Stream, MainParam_t *mp); 
-int unix_loop(UNUSED(Stream_t *Stream), MainParam_t *mp, char *arg,
+int unix_loop(Stream_t *Stream UNUSEDP, MainParam_t *mp, char *arg,
 	      int follow_dir_link);
 
 static int _unix_loop(Stream_t *Dir, MainParam_t *mp,
@@ -95,7 +95,7 @@ static int _unix_loop(Stream_t *Dir, MainParam_t *mp,
 	return unix_dir_loop(Dir, mp);
 }
 
-int unix_loop(UNUSED(Stream_t *Stream), MainParam_t *mp,
+int unix_loop(Stream_t *Stream UNUSEDP, MainParam_t *mp,
 	      char *arg, int follow_dir_link)
 {
 	int ret;
